skyuxconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "skyux-media",
```

Comment:
We should add the `"$schema"` property here, which will give the file type support in the IDE.
```
"$schema": "./node_modules/@skyux/config/skyuxconfig-schema.json",
```